src/common/com/intellij/plugins/haxe/util/HaxeFileUtil.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2017 Eric Bishton
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.intellij.plugins.haxe.util;
+
+import com.intellij.openapi.vfs.VirtualFile;
+import com.intellij.openapi.vfs.VirtualFileManager;
+
+import java.io.IOException;
+
+/**
+ * Created by ebishton on 8/10/17.
```

Comment:
Yeah.  IDEA puts those in automatically when you create a class.  I used to hate them, but I've been finding it very useful recently to see who originally created a class (without going through git), because it's a hint as to _why_ it was added.  I have enough of the project's context in my head that I know who was working on what at what time, so that the authorship is useful to me.